Factory/RequestFactory.php

```diff
@@ -50,6 +52,12 @@ public function createOne(array $tokens, $apiUrl, array $options) {
         $query   = HttpQuery::create($tokens);
         $payload = $method === HttpMethods::GET || $method === HttpMethods::DELETE ? null : Payload::create($tokens);
 
+        $options['CURLOPT_HTTPHEADER'] = array_merge(
```

Comment:
Hi! Thanks for reviewing the code.

I'm a bit confused about this line. Or did you mean that the Input is the $tokens?

> Let's build a new type HttpHeader. Input is the options array. Output is the options array containing just one entry with the key CURLOPT_HTTPHEADER.